fem/bilininteg.cpp

```diff
@@ -659,6 +659,9 @@ void MixedScalarVectorIntegrator::AssembleElementMatrix2(
    MFEM_ASSERT(this->VerifyFiniteElementTypes(trial_fe, test_fe),
                this->FiniteElementTypeFailureMessage());
 
+   MFEM_VERIFY(VQ, "MixedScalarVectorIntegrator: "
+               "VectorCoefficient must be set");
```

Comment:
Not big deal but this check seems unnecessary: `VQ` is initialized from the address of a reference which is never supposed to be `NULL`.